src/code/audio_load.c

```diff
@@ -890,32 +886,26 @@ void AudioLoad_RelocateFont(s32 fontId, SoundFontData* fontDataStartAddr, Sample
     // Starting from the 3rd u32 in fontData is the list of offsets to the instruments
     // Loop through the instruments
     for (i = 2; i <= 2 + numInstruments - 1; i++) {
-        // Some instrument data entries are empty, represented by an offset of 0 in the instrument offset list
-        if (fontData[i] != NULL) {
-            // Relocate the instrument offset (relative to the start of the font data) to a pointer (a ram address)
+        // Some instrument data entries are empty, represented by an offset of 0 in the list of instrument offsets
+        if ((u32)fontData[i] != 0) {
```

Comment:
What's the motivation for switching `(fontData[i] != NULL)` to `((u32)fontData[i] != 0)`?

Reply:
it's an offset of 0 at this point (before relocation), not a NULL pointer (after relocation)
but it's mostly arbitrary tbh